yarn.lock

```diff
@@ -1472,6 +1472,11 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.14.0.tgz#1483caa1dd4e7097ac48a5872e93d7f896e8402c"
   integrity sha512-WNNVTMpRir6a2Voa0yu3BZJmy2w7fofrokL+nyFQM40bACtX68KDRDE96HA5W5+4q8rEaNSXV3DzSzLXm6ekfA==
 
+"@fluentui-react-native/theme-types@0.19.1":
```

Comment:
I think you should double-check your versions again, this package shouldn't need to be added to our yarn.lock since it should be part of the workspace.

Reply:
There were conflicts, I got all the versions from main. But I'll check again